app/controllers/pending-user-updates.js

```diff
@@ -49,7 +49,7 @@ export default Controller.extend({
     });
   }),
 
-  displayedUpdates: computed('allUpdates.@each.user.fullName', 'filter', 'offset', 'limit', function(){
+  displayedUpdates: computed('allUpdates.@each.user.fullName', 'filter', 'offset', 'limit', 'deletedUpdates.[]',  function(){
```

Comment:
`allUpdates.@each.user.fullName` is also invalid and may be partially to blame for this issue.  `@each` doesn't work on nested properties.  Should be changed to `allUpdates.@each.user` and then see if we need the deletedUpdates watcher.